web/tests/lynxkite.ts

```diff
@@ -1120,25 +1076,15 @@ function sortHistogramValues(values) {
   });
 }
 
-// A promise of the list of error messages.
-function errors() {
-  return $$('.top-alert-message').map(function (e) {
-    return e.getText();
-  });
+export function errors(page: Page) {
```

Comment:
So do we need to support it in all the other tests?

Reply:
> So do we need to support it in all the other tests?

You mean like check that there are no errors? I don't think that's necessary. In this test we're checking that there _is_ an error.